middleware/index.ts

```diff
@@ -190,11 +190,7 @@ function createContext(id: string, logger: Logger, reqContext?: any, accessToken
  * @return secret value if successfully loaded, undefined if not found/loaded.
  */
 function getSecret(name: string, key: string, logger: Logger) : string|undefined {
-    const secretMap = new Secrets(logger).get(name);
-    if (secretMap && key in secretMap) {
-        return secretMap[key];
-    }
-    return undefined;
+    return new Secrets(logger).getValue(name, key);
```

Comment:
@dhagberg-sf Should we use the same Secret for both retrieving the sf-debug secret as well as using it to init the context later?

Reply:
Yes we should reuse same Secrets object - fixed.